src/platform/services-linux.c

```diff
@@ -34,13 +34,14 @@
  *
  * Linux implemention of services.
  */
-#ifdef __linux__
 
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
 
+#if defined(__linux__) || defined(__GLIBC__)
```

Comment:
Is there a reason to move the check down? I'd like it to be at the very top because this file is linux/glibc specific. But I'm being picky, I'll move it back at the top after merging.